### ISSUE TITLE
Install

### ISSUE THREAD

**Issue by capriciousduck** [tdulcet/Colab-Autorun-and-Connect#1]
How do I install the extension on firefox?

**Comment by tdulcet** [comment]
Download it with this command:
```sh
git clone --recurse-submodules https://github.com/tdulcet/Colab-Autorun-and-Connect.git
```

Then follow [these instructions](https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/) to install it in Firefox.

**Comment by capriciousduck** [comment]
Ok. How do I install this on Chrome?

**Comment by tdulcet** [comment]
1. The same step from above to download it.
2. As instructed on the README:
    1. Download the [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) (specifically download the `browser-polyfill.js` file from [here](https://unpkg.com/webextension-polyfill/dist/) to your directory from step 1. above).
    2. Rename the `chromemanifest.json` file to `manifest.json`.  
3. Then follow [these instructions](https://developer.chrome.com/docs/extensions/mv3/getstarted/#manifest) (steps 1-3 in the middle of the page) to install it in Chrome/Chromium.

**Comment by 19PHOBOSS98** [comment]
Hi, could you please help me :) I tried installing it in chrome and it had some errors in the extension manager:

<img width="668" alt="Screen Shot 2021-05-14 at 12 04 06 PM" src="https://user-images.githubusercontent.com/37253663/118219677-8f827c00-b4ac-11eb-996f-e245bff26ce4.png">
<img width="655" alt="Screen Shot 2021-05-14 at 12 04 12 PM" src="https://user-images.githubusercontent.com/37253663/118219690-9610f380-b4ac-11eb-98cf-92eaa0e02923.png">

I cloned it using the link above
I downloaded `browser-polyfill.js` from the given [link](https://unpkg.com/webextension-polyfill/dist/)
the cloned folder came with another `manifest.json` file so I got rid of that before renaming the `chromemanifest.json` file into `manifest.json`
I then opened up the extension manager and enabled developer mode

what am I doing wrong?

**Comment by tdulcet** [comment]
> I downloaded `browser-polyfill.js` from the given [link](https://unpkg.com/webextension-polyfill/dist/)

You need to first click on the “View Raw” button in the top right corner of the page before downloading the file, so that you download just the JavaScript file and not the whole HTML page. Unfortunately, that page is not very intuitive. 😕

**Comment by 19PHOBOSS98** [comment]
ok, so I hit "View Raw" as you said and right clicked the page and clicked on "Save As" and manually dropped the new `browser-polyfill.js` into the `Colab-Autorun-and-Connect` folder and it gave me this error at the extension manager:
<img width="688" alt="Screen Shot 2021-05-14 at 2 01 13 PM" src="https://user-images.githubusercontent.com/37253663/118228523-edb75b00-b4bc-11eb-9bc8-b0b70c6fee97.png">
<img width="684" alt="Screen Shot 2021-05-14 at 2 01 32 PM" src="https://user-images.githubusercontent.com/37253663/118228542-f445d280-b4bc-11eb-8f80-934680b6f34b.png">

I tried manually copy-pasting the whole script into a text file and saving it as .js. It still gave me the same error :(

**Comment by tdulcet** [comment]
> it gave me this error at the extension manager:

Those are just warnings from one of the libraries that this extension uses. The first one is erroneous (see https://github.com/TinyWebEx/AddonSettings/issues/7). You can safely ignore them all.

**Comment by 19PHOBOSS98** [comment]
Oh ok, cool. Thanks!

**Comment by tdulcet** [comment]
For anyone following this issue, you can now install the extension in Firefox directly from AMO: https://addons.mozilla.org/en-US/firefox/addon/colab-autorun-and-connect/

**Comment by kylevessPL** [comment]
Since 2023, manifest v2 has been deprecated and migration to v3 should be done beforehand:
https://developer.chrome.com/docs/extensions/migrating

**Comment by tdulcet** [comment]
Yes, MV2 may be deprecated in Chrome, but should still work just fine when sideloading extensions. 😉

PRs are of course welcome to migrate to MV3...
